mainnet/13-wysiwyg-finance-transfer/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "voting": "0x0e835020497b2cd716369f8fc713fb7bd0a22dbf",
+  "submitterJustification": "If approved, this proposal will transfer 1 ETH to the Aragon Association multisig address 0xcafe1a77e84698c83ca8931f54a755176ef75f2c",
```

Comment:
I was always assuming that these would be IPFS links as well, but it's a good point that that would make it harder for us to load these votes in a card / etc. We anticipate these to be small enough to be sent in original form on-chain?

Reply:
True, but I wasn't thinking about that honestly, my intention was to have a mixed situation like plain text for the submitter but an IPFS file for the challenger. But we should support having both using IPFS

Reply:
If we're not expecting these to be very long, I think it's acceptable to keep them as plaintext for now. Loading 20/30 files, even if small, would be kind of a pain for the frontend (it's why we don't do it for the court dashboard at the moment).